### PR TITLE
ExpApproach updates

### PR DESCRIPTION
**Pull request by UlrikeS91**
All approaches have an InterLex ID now. A few comments/questions for @lzehl and @tgbugs:

1) I couldn't find "chemogenetics". Does this exist yet?

2) "genetics" and "informatics" are listed under "scientific discipline" in InterLex. Will those get a new/updated entry or is this was we should use here?

3) "optogenetics" exists twice (http://uri.interlex.org/ilx_0108079 &  http://uri.interlex.org/ilx_0737732). Definitions were rather similar, so I just picked one. Any objections? 

4) The "preferred ID" was the InterLex ID. So I copied the same one in for the "interlexID" as well. In almost all cases this was the preferred ID according to the InterLex entry. @lzehl if you don't want this repetition, we can remove them from the preferred ID again.

### CODE REVIEW

**Comment by tgbugs** [comment]
There are a number of issues here that I need to work through here related to the identifiers. I'm not sure whether you should use the existing terms for genetics and optogenetics because their modelling is different and we will need to figure out how to deal with merging them.

I would suggest using the `mod:` as the preferred ids since they are what is used in the ontology and they now resolve! (yay! http://uri.interlex.org/tgbugs/uris/readable/modality/Anatomy).
https://github.com/SciCrunch/NIF-Ontology/blob/methods/ttl/approach.ttl
I have created mod:Genetics and friends but we need to get them interlex ids (@tmsincomb) so that they will resolve. We will update the InterLex records to reflect this now that they resolve correctly.

For reference, check the children tab [here](https://scicrunch.org/scicrunch/interlex/view/ilx_0770148#children) for the list of the actual approach terms that is currently in InterLex.

**Comment by UlrikeS91** [comment]
Thanks, @tgbugs. I've made some adjustments according to your feedback.

1. I removed interlex ID, preferred ID and synonyms from "chemogenetics", "genetics", "informatics" and "optogenetics". These are the only four that aren't part of the experimental modalities (https://scicrunch.org/scicrunch/interlex/view/ilx_0770148) yet. As soon as they are added and/or get interlex IDs, we can update those again. 
2. For the rest, I changed the preferred ID to "http://uri.interlex.org/tgbugs/uris/readable/modality/<<name>>".

@lzehl, this PR should be ok to merge now (but decide for yourself :P).

File: instances/experimentalApproach/electrophysiology.jsonld [hunk 1]
```diff
@@ -6,9 +6,9 @@
   "@type": "https://openminds.ebrains.eu/controlledTerms/ExperimentalApproach",
   "definition": "Any experimental approach focused on electrical phenomena associated with living systems, most notably the nervous system, cardiac system, and musculoskeletal system.",
   "description": null,
-  "interlexIdentifier": null,
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0741202",
   "knowledgeSpaceLink": null,
   "name": "electrophysiology",
-  "preferredOntologyIdentifier": "http://uri.interlex.org/base/ilx_0741202",
+  "preferredOntologyIdentifier": "	http://uri.interlex.org/tgbugs/uris/readable/modality/Electrophysiology",
```
Comment (lzehl) [inline instances/experimentalApproach/electrophysiology.jsonld]: I think there are too many spaces here

**Review by lzehl — CHANGES_REQUESTED** [review]
except one (I think) typo it looks good.

File: instances/experimentalApproach/electrophysiology.jsonld (hunk 1 above)
Comment (UlrikeS91) [reply instances/experimentalApproach/electrophysiology.jsonld]: Yes, happened while copy-pasting. Fixed it.

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
@UlrikeS91 also here: could you transfer this please also to the controlledTerms repo? The openMINDS_instances repo will be deprecated. Instances are version dependent and are therefore now stored on the respective version branch of the openMINDS metadata model repo.

**Comment by UlrikeS91** [comment]
> @UlrikeS91 also here: could you transfer this please also to the controlledTerms repo? The openMINDS_instances repo will be deprecated. Instances are version dependent and are therefore now stored on the respective version branch of the openMINDS metadata model repo.

Done :)

**Pull request merged**